src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2683,7 +2763,11 @@ Mavlink::start(int argc, char *argv[])
 	px4_task_spawn_cmd(buf,
 			   SCHED_DEFAULT,
 			   SCHED_PRIORITY_DEFAULT,
+#ifdef CONFIG_NET
```

Comment:
Same as comment as below - in general where possible 1 ifdef to configure the deltas and then no ifdefs in the code is a cleaner approach.